examples/miscellaneous/parallel_backend_benchmarks.py

```diff
@@ -0,0 +1,197 @@
+"""
+=======================================================================
+Comparing parallelization with context manager using different backends
+=======================================================================
+
+In this notebook we demo how to use a context manager to change the default
+backend used by the joblib implementation inside any scikit-learn object that
+has a parameter `n_jobs`.
+
+In practice, whether increasing the number of workers is helpful at improving
+runtime depends on many factors. It is usually better to experiment rather than
+assuming that it is always a good thing. In some cases it can be highly
+detrimental to performance to run multiple copies of some estimators or
+functions in parallel (see :ref:`oversubscription`).
+
+To run this notebook locally, you may need to install dask and ray. These
+packages can be installed with `pip install dask-ml "ray[default]"`.
```

Comment:
Is `dask-ml` required or does this example only need `dask`?

Reply:
In my experiment, `dask` was enough